py/client/pydeephaven/session.py

```diff
@@ -519,3 +536,7 @@ def input_table(self, schema: pa.Schema = None, init_table: Table = None,
         input_table = self.table_service.grpc_table_op(None, table_op, table_class=InputTable)
         input_table.key_cols = key_cols
         return input_table
+
+    def plugin_client(self, exportable: Any) -> PluginClient:
+        """Part of the experimental plugin API."""
```

Comment:
The doc string wasn't super helpful.  It read as "this function is part of the API".